images/Dockerfile.kairos

```diff
@@ -0,0 +1,86 @@
+# WARNING: Do not build this image on its own, use the different Dockerfile.kairos-* instead
+FROM quay.io/kairos/framework:${FRAMEWORK_VERSION}_${SECURITY_PROFILE} AS framework
+
+FROM all AS base-kairos
+
+# Symlinks to make elemental installer work
+RUN ln -sf /usr/sbin/grub-install /usr/sbin/grub2-install
+RUN ln -sf /usr/bin/grub-editenv /usr/bin/grub2-editenv
+
+# Originally done in Earthly
+COPY --from=framework / /
+
+RUN rm -rf /etc/ssh/ssh_host_*
+
+COPY naming.sh /usr/local/bin/naming.sh
+
+RUN sed -i -n '/KAIROS_/!p' /etc/os-release
+
+# need to be defined after FROM for them to be replaced in the RUN bellow
+ARG FLAVOR
+ARG FLAVOR_RELEASE
+ARG VARIANT
+ARG OS_ID=kairos
+ARG HOME_URL="https://github.com/kairos-io/kairos"
+ARG BUG_REPORT_URL="https://github.com/kairos-io/kairos/issues"
+ARG GITHUB_REPO="kairos-io/kairos"
+# TODO: merge these
+ARG VERSION
+ARG FAMILY
+ARG MODEL
+ARG REGISTRY_AND_ORG="quay.io/kairos"
+ARG K3S_VERSION
+ARG TARGETARCH
+ARG OS_NAME=kairos-${VARIANT}-${FLAVOR}-${FLAVOR_RELEASE}
+ENV KAIROS_VERSION="${VERSION}${K3S_VERSION:+-k3s$K3S_VERSION}"
+ENV OS_VERSION=${KAIROS_VERSION}
+ENV OS_LABEL=${KAIROS_VERSION}
+RUN OS_LABEL=$(naming.sh container_artifact_label) \
+    OS_REPO=$(naming.sh container_artifact_repo) \
+    ARTIFACT=$(naming.sh bootable_artifact_name) \
+    envsubst >>/etc/os-release </usr/lib/os-release.tmpl
+RUN naming.sh container_artifact_name > /IMAGE
+
+# TODO no benefit of keaping it?
+# RUN rm /usr/local/bin/naming.sh
```

Comment:
The `kairos-agent` will soon have this functionality: https://github.com/kairos-io/kairos/issues/1999#issuecomment-1830264515

so we won't need any additional scripts.